rotkehlchen/tasks/calendar.py

```diff
@@ -178,20 +180,110 @@ def maybe_create_ens_reminders(database: DBHandler) -> None:
+            (user_address := string_to_evm_address(crv_event.location_label)) in blockchain_accounts.get(  # noqa: E501
+                blockchain := ChainID.deserialize(crv_event.location.to_chain_id()).to_blockchain(),  # noqa: E501
+            ) and (locktime := Timestamp(locktime)) > current_ts  # those ending in the future
+        ):
+            continue
+
+        calendar_entry_description = f'Lock period for {crv_event.balance.amount} CRV in vote escrow ends on {customizable_date.timestamp_to_date(locktime)}'  # noqa: E501
+        if (db_calendar.query_calendar_entry(
+                filter_query=CalendarFilterQuery.make(
+                    and_op=True,
+                    name='CRV vote escrow lock period ends',
+                    addresses=[OptionalBlockchainAddress(
+                        blockchain=blockchain,
+                        address=user_address,
+                    )],
+                    blockchain=blockchain,
+                    counterparty=CPT_CURVE,
+                    description=calendar_entry_description,
+                ),
+        ))['entries_found'] == 0:  # if calendar entry doesn't exist, add it
+            crv_calender_entries.append(db_calendar.create_calendar_entry(CalendarEntry(
+                name='CRV vote escrow lock period ends',
+                timestamp=locktime,
+                description=calendar_entry_description,
+                counterparty=CPT_CURVE,
+                address=user_address,
+                blockchain=blockchain,
+                color=CRV_CALENDAR_COLOR,
+                auto_delete=True,
+            )))
+
+    _, failed_to_add = db_calendar.create_reminder_entries(reminders=[
+        ReminderEntry(
+            identifier=calendar_identifier,  # this is only used for logging below, it's auto generated in db  # noqa: E501
+            event_id=calendar_identifier,
+            secs_before=0,
         )
+        for calendar_identifier in crv_calender_entries
+    ])
 
     if len(failed_to_add) == 0:
         return
 
     log.error(  # failed_to_add is a list of calendar_identifier that were passed as identifiers of reminder entry above  # noqa: E501
-        f"""Failed to add the ENS expiry reminders for {', '.join([
+        f"""Failed to add the CRV lock period end reminders for {', '.join([
             entry.name for entry in db_calendar.query_calendar_entry(CalendarFilterQuery.make(
                 identifiers=failed_to_add,
             ))['entries']
         ])}""",
     )
+
+
+def create_all_reminders(database: DBHandler) -> None:
+    """Create all needed calendar reminders"""
+    maybe_create_ens_reminders(database)
+    maybe_create_locked_crv_reminders(database)
+
+    with database.conn.write_ctx() as write_cursor:
+        database.set_static_cache(
+            write_cursor=write_cursor,
+            name=DBCacheStatic.LAST_CREATE_REMINDER_CHECK_TS,
+            value=ts_now(),
```

Comment:
it probably makes more sense to save the same `ts_now()` value into a variable here and pass/use that in the above functions. So that there is no time difference between when we check and when we save the timestamp.